packages/client/examples/node.js/put-car-dag-cbor.js

```diff
@@ -0,0 +1,106 @@
+    return
+  }
+
+  const storage = new Web3Storage({ token })
+  await simpleCborExample(storage)
+  await cborLinkExample(storage)
+}
+
+main()
+
+/**
+ * Stores a simple "hello world" object as IPLD data, encoded using dag-cbor.
+ * @param {Web3.Storage} storage a Web3.Storage client
+ */
+async function simpleCborExample (storage) {
+  // encode the value into an IPLD block and store with Web3.Storage
+  const block = await encodeCborBlock({ hello: 'world' })
+  const car = await makeCar(block.cid, [block])
+
+  // upload to Web3.Storage using putCar
+  console.log('🤖 Storing simple CBOR object...')
+  const cid = await storage.putCar(car)
+  console.log(`🎉 Done storing simple CBOR object. CID: ${cid}`)
+  console.log(`💡 If you have ipfs installed, try: ipfs dag get ${cid}\n`)
+}
+
+/**
+ * Stores a CBOR object that links to another CBOR object by CID.
+ * @param {Web3Storage} storage a Web3.Storage client
+ */
+async function cborLinkExample (storage) {
+  // Encode a simple object to get its CID
+  const addressBlock = await encodeCborBlock({ email: 'zaphod@beeblebrox.galaxy' })
+
+  // Now we can use the CID to link to the object from another object
+  const personBlock = await encodeCborBlock({
+    title: 'Galactic President',
+    description: 'Just this guy, you know?',
+    contact: addressBlock.cid
+  })
+
+  // pack everything into a CAR
+  const car = await makeCar(personBlock.cid, [personBlock, addressBlock])
+
+  // upload to Web3.Storage using putCar
+  console.log('🤖 Storing CBOR objects with CID links between them...')
+  const cid = await storage.putCar(car)
+  console.log('🎉 Stored linked data using dag-cbor. Root CID:', cid)
+  console.log(`💡 If you have ipfs installed, try: ipfs dag get ${cid}`)
+  console.log(`🔗 You can also traverse the link by path: ipfs dag get ${cid}/contact\n`)
+}
+
+/**
+ * Encodes an object into an IPLD block using the dag-cbor codec.
+ * @param {any} value - any JS value that can be converted to CBOR (if it can be JSON.stringified, it will work)
+ * @returns {Block} a block of encoded IPLD data.
+ */
+async function encodeCborBlock (value) {
+  return encode({ value, codec: cbor, hasher: sha256 })
+}
+
+/**
+ * Takes a root CID and an iterable of encoded IPLD blocks, and returns a CarReader that
+ * can be used with Web3.Storage.putCar
+ * @param {string} rootCID the CID of the root block of the IPLD graph
+ * @param {Iterable<Block>} ipldBlocks a collection of encoded IPLD blocks
+ * @returns {CarReader} a CarReader for sending the CAR data to Web3.Storage
+ */
+async function makeCar (rootCID, ipldBlocks) {
+  // create a new CarWriter, with the given root CID
+  const { writer, out } = CarWriter.create([rootCID])
+
+  // add the blocks to the CAR and close it
+  for (const block of ipldBlocks) {
+    writer.put(block)
+  }
+  writer.close()
+
+  // create a new CarReader we can hand to Web3.Storage.putCar
+  const reader = await CarReader.fromIterable(out)
+  return reader
+}
```

Comment:
```suggestion
async function makeCar (rootCID, ipldBlocks) {
  return new CarReader(1, [rootCID], ipldBlocks)
}
```

...or since it's a 1 liner just inline this function?

I'm not 100% sure this is better since it doesn't create a CAR file...but it's certainly less code.